changelogs/fragments/23-hashi-vault-lookup-refresh.yaml

```diff
@@ -0,0 +1,13 @@
+bugfixes:
```

Comment:
These changelog entries are a mix of `bugfixes` and `minor_changes`, and should be separated out accordingly.

Reply:
@flowerysong 
imo all 3 of the things listed under `bugfixes` are in fact bugfixes
Which one(s) do you think should be moved to `minor_changes`?

Reply:
I think @flowerysong means that there should be two separate PRs, one for bugfixes, one for features.

Reply:
No, I somehow completely missed that you already had them split into two lists, and thought you had put everything in the `bugfixes` list.

Reply:
Oh ok, then it should be fine :)